ac/ac-imgClass/src/imgClassifier.jsx

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import React from 'react';
+import Mousetrap from 'mousetrap';
+import type { ActivityRunnerT } from 'frog-utils';
+import styled from 'styled-components';
+
+const ImgBis = styled.img`
+  max-width: 80%;
+  max-height: 100%;
+  position: relative;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+`;
+
+const FlexDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  height: 90%;
+`;
+
+const ImgPanel = (props: { url: string }) =>
+  <div style={{ width: '90%', height: '100%' }}>
+    <ImgBis alt="" src={props.url} />{' '}
+  </div>;
+
+const ShortCutPanel = (props: { categories: Array<string> }) =>
```

Comment:
Two UI requests:
- Could this short cut panel be centered vertically in its parent's div, so it will be at the same level as the image?
- My first reflex was to click on the shortcuts, but it did nothing. I believe it would be nice to have them clickable and user would chose whether to use the short cuts or just click. what do you think?


![screenshot from 2017-08-17 11-10-17](https://user-images.githubusercontent.com/8818081/29404939-c3956fca-833c-11e7-901e-73495426146b.png)